news/_posts/2020-05-07-productboard-portal-launch.md

```diff
@@ -0,0 +1,100 @@
+
+#### Under consideration
+Lists all topics and some features that are already well structured and identified. At this step rating a card and telling the team why it is important will help the team prioritize it.
+
+### User research
+Lists all the features the product team has decided to work on at the moment. When the team starts working on a feature, the first step is always to conduct some user research to understand which types of users need this feature, what are the use cases around it, and what’s its purpose.
+The goal of the user research is to identify the problem or need, to solve.
+
+#### Design
+Once the problem is identified, the designers start working on the best solution to solve it.
+This tab lists all the features going under design.
+
+
+### Comment and vote for existing ideas
+When clicking on a card you can see 2 fields at its bottom. One to comment, and one to vote on the feature's impact **on you**.
+
+Voting for ideas is really important to help the team.
+Here is what happens behind the scenes:
+
+![PrestaShop Productboard user score](/assets/images/2020/05/productboard-user-score.png)
+
+When rating the impact of a feature, Productboard adds your vote to a “user score” that the team takes directly into account when evaluating features through its prioritization score.
+The more an idea is voted, the better rank it gets, and the more probability it has to integrate the software.
+
+This “user score” is not the only indicator that the team takes into account when evaluating a feature but it is key. Complexity, innovation, business impact for the merchants, legal aspects, are some of the criteria studied as well.
+
+<div class="alert alert-note" role="alert">
+Of course, the goal is not to vote “Critical” on every feature idea. This would cancel the benefits of the open votes.</div>
+
+Now you might also wonder what happens when you add a comment.
+Well, the comment is automatically associated with the card in the team interface.
+It helps the product team, the designers and the developers precisely identify the users’ needs, and thus design the feature the best way possible to answer these needs.
+
+In the end, it is all about delivering as much value as possible to the users, with all kinds of users in mind.
+
+### Submit a new idea
+In addition to seeing all the features ideas, commenting and voting, you can submit new ones.
+
+A “Submit an idea” button is located in the Portal’s top-right corner. Click on it and follow the steps.
+
+![How to sumbit a new idea on PrestaShop Feedback](/assets/images/2020/05/productboard-new-idea.gif)
+
+Here again you can indicate the importance of this insight for you.
+
+Emails addresses are required by default by Productboard to avoid spamming.
+The team might use it to contact you about your idea as well.
+
+
+## What’s next
+This is just the beginning, and the product team still needs some time to get used to the tool and to adapt its product development methodology.
```

Comment:
```suggestion
## Know what is next

Once the problem exposed, the solution defined and the design approved, the feature enters its development phase. For the Core product team, the next steps are writing specifications on the [dedicated repository](https://github.com/PrestaShop/prestashop-specs) and creating all related issues so that the Core dev team or the community can have all the necessary elements to develop them. All those steps are lead on GitHub, Productboard is just the tool that allows the Core product team to collect feedback and prioritize features.
```